go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/btcsuite/btcwallet
 
 require (
-	github.com/btcsuite/btcd v0.23.5-0.20230711222809-7faa9b266231
+	github.com/btcsuite/btcd v0.23.5-0.20231215221805-96c9fd8078fd
```

Comment:
think we can update now (unless you wanna wait for a tag on that side)

Reply:
cool updated!